### ISSUE TITLE
Make TPAC more welcoming

### ISSUE THREAD

**Issue by TzviyaSiegman** [w3c/PWETF#48]
Provide advice to W3M and AB about how to make TPAC a more welcome environment

**Comment by TzviyaSiegman** [comment]
* Every WG session starts with intros, CEPC, overview of group dynamic
* Consider other approaches to un-conference on plenary day
* More networking
* Translator tools

**Comment by jorydotcom** [comment]
We had a great conversation at dinner last night of the AC meeting about this (Dan A, Natasha, Theresa, David Singer, David Barron, Alan Stearns) - here's one thing I can recall:

* Have 2 separate 'training' meetings at the same time, 1 for newcomers and 1 for chairs/meeting leaders. The newcomers meeting covers things like how to use IRC and reinforces the importance of their contributions (encourages them to speak up); the chairs meeting covers CEPC implementation, meeting facilitation tips, etc. The idea is everyone gets "training" for how to make the most out of TPAC but it's tailored a bit.

**Comment by torgo** [comment]
I will note that the current TPAC page (for Fukuoka) does not include a link to or even a mention of a code of conduct or diversity statement for the event. This would be a good first step.

**Comment by nigelmegitt** [comment]
@torgo I saw a link to the CEPC from the "Participation rules" section of the draft TPAC Fukuoka page. Not sharing the link here because I don't think it's been published yet.

**Comment by chaals** [comment]
Yes, let's please consider a more friendly approach to planning sessions than an hour in which a dozen or so people fight over a pinboard, and a few hundred sit on the sidelines and read email for an hour.

(The people at the pinboard are often proposing something for every session. If they did it in advance, maybe we could get another useful session in the day).

I don't know of great software for this, but a form based on a properly-constructed HTML table (something more accessible than wiki table markup) might be a reasonable starting point.

**Comment by chaals** [comment]
Yes to introductory sessions. I would prefer to have the chairs meeting on the Sunday instead of an early-morning breakfast. And putting the chairs and new members *together* for a chunk of it makes sense too. (Note the cost implications - do we have a venue available on Sunday?)

**Comment by wareid** [comment]
Introductory sessions sound good, something directed at members and then chairs, overlapping them to have them meet is good too. 

I have tried this in our working group but put some emphasis on chairs checking in with new members or those that aren't participating as much just to make sure they are comfortable in the group/don't have any issues or questions preventing them from participating. Sometimes people are uncomfortable speaking up if they don't know if they completely understand the material, but they often ask the best questions.

**Comment by jnurthen** [comment]
Time zones would make doing anything on Sunday challenging. I don’t think you would get good chair attendance.

**Comment by nigelmegitt** [comment]
Looking back over several TPACs, I would usually be able to make a Sunday meeting but prefer not to - that's my time zone adjustment day usually. This year I certainly will not be able to attend on the Sunday. 

@chaals I think those of us who have attended the Chairs breakfast over a number of years do understand (!) that you would prefer a different time, but so far it hasn't ever stopped you from attending - in the spirit of consensus, would it be fair to say that, though you don't like it, you can in fact live with it?

**Comment by torgo** [comment]
One note on the idea of a Sunday meeting: like many other w3c member organizations, Samsung has people who participate in W3C working groups and efforts who are actually located in many spots around the world. TPAC is one opportunity to bring those people together - which we plan to do on the Sunday before the next TPAC in a bit of an internal workshop. I know other organizations have done the same sort of thing in the past - so scheduling an intro session on the Sunday might not be such a good idea. How about an intro session over video call a couple of weeks before the meeting, which could be run in a few different time zones? We took this approach in the JS Foundation / Node Foundation merger discussion by running a number of town hall style video conferences which were also put up on YouTube... Different context, but we could take the same approach.

**Comment by hober** [comment]
> * Every WG session starts with intros, CEPC, overview of group dynamic

When kicking off intros, chairs should lead by example in providing their own name and pronouns, and should encourage everyone to provide pronouns as they introduce themselves.

All badges should have pronouns on them too!

**Comment by TzviyaSiegman** [comment]
@hober As long as providing pronouns is optional I agree. I have heard some devastating stories about pronouns stickers.

**Comment by LJWatson** [comment]
I like the idea of holding a chairs meeting on the Sunday afternoon. I don't mind the timing of the Tuesday breakfast (though could cheerfully do without it), but I don't think it's enough.

We say hello, introduce ourselves, exchange some news and a few useful tips, and that's fine if you're an existing chair who wants to stay in touch with other chairs.

From my own experience, and based on many conversations over recent months, I realise that many chairs have never been properly introduced to the W3C Process, to how the Rec Track works, to where horizontal review fits in. Others do not know about the chair mentoring programme, about the code of conduct and/or the ombudspeople, how to deal with difficulties within their WG etc.

We could use the Sunday afternoon to provide some introductory chair training for new and existing chairs who might be interested.

**Comment by LJWatson** [comment]
Could we create a TPAC buddy scheme for first timers, as we do for the AC meetings?

Could we put our collective minds to work on solving the accessibility problem of badges too? :)

**Comment by LJWatson** [comment]
We could do something like the "dinner with new friends" that I've seen at web conferences. You can put your name down if you're attending the conference and don't know anyone else. People are then assigned to groups of five/six others, and are given a reservation at a local restaurant.

**Comment by AdaRoseCannon** [comment]
A couple of people have commented that they don't understand what the technical plenary day means: https://twitter.com/AmeliasBrain/status/1139978462897115136?s=20

**Comment by LJWatson** [comment]
@AdaRoseCannon commented:
>A couple of people have commented that they don't understand what the technical plenary day means>

Looking back to the early TPACs, it seems the Wednesday did follow a much more typical plenary model. The [2001 plenary agenda](https://www.w3.org/2001/02/Plenary/Agenda.html) is apparently much more formal, and more clearly curated than the model we have now for example.

These days it is much more like an unconference, and that is a term more likely to be familiar and/or meaningful to most W3C participants I suspect.

Since there is more to TPAC than its acronym really conveys (Technical Plenary and Advisory Committee), perhaps referencing the Wednesday as "Unconference day" as opposed to WG/CG meeting days" might help?

Also including a short intro somewhere on the TPAC schedule/information page might help. For example:

"The technical plenary (unconference) day is an opportunity for everyone to propose, choose, and participate in sessions on a wide variety of topics relating to W3C activities. It is a chance to contribute to discussions and connect with people working on ideas that may not fall within your usual areas of activity."

Or something!

**Comment by TzviyaSiegman** [comment]
>  "Since there is more to TPAC than its acronym really conveys (Technical Plenary and Advisory Committee), perhaps referencing the Wednesday as "Unconference day" as opposed to WG/CG meeting days" might help?
> 
> Also including a short intro somewhere on the TPAC schedule/information page might help. For example:
> 
> "The technical plenary (unconference) day is an opportunity for everyone to propose, choose, and participate in sessions on a wide variety of topics relating to W3C activities. It is a chance to contribute to discussions and connect with people working on ideas that may not fall within your usual areas of activity."

I agree with @LJWatson's suggestion. cc/@koalie

**Comment by koalie** [comment]
> I will note that the current TPAC page (for Fukuoka) does not include a link to or even a mention of a code of conduct or diversity statement for the event. This would be a good first step. [update: i do see that it's linked prominently under "participation rules" but I still think it should be directly linked from the front page at least from the bottom links]

In the meantime this made it to the front page; it's the first thing you see under the top banner at https://www.w3.org/2019/09/TPAC/

**Comment by koalie** [comment]
> > "Since there is more to TPAC than its acronym really conveys (Technical Plenary and Advisory Committee), perhaps referencing the Wednesday as "Unconference day" as opposed to WG/CG meeting days" might help?
> > Also including a short intro somewhere on the TPAC schedule/information page might help. For example:
> > "The technical plenary (unconference) day is an opportunity for everyone to propose, choose, and participate in sessions on a wide variety of topics relating to W3C activities. It is a chance to contribute to discussions and connect with people working on ideas that may not fall within your usual areas of activity."
> 
> I agree with @LJWatson's suggestion. cc/@koalie

Done!
cf. https://www.w3.org/2019/09/TPAC/participation.html#plenary
https://www.w3.org/2019/09/TPAC/schedule.html#Wednesday

**Comment by koalie** [comment]
I see this issue is still open.
Here's a status:
* our code of ethics and professional conduct has been made prominent
* the team is going to experiment with a "no scramble" approach to the breakout sessions planning (see https://www.w3.org/wiki/TPAC/2019/FAQ#How_does_the_agenda_get_built.3F for detail)
* I do not plan to host a "welcome/orientation" session on the Sunday
* I do not plan to create the "warmup cards" slide deck I had come up with last year, because we changed our TPAC website to make it more usable.
